apps/minifront/src/state/dutch-auction/assemble-schedule-request.test.ts

```diff
@@ -52,48 +61,9 @@ const ARGS: Parameters<typeof assembleScheduleRequest>[0] = {
 };
 
 describe('assembleScheduleRequest()', () => {
-  it('correctly converts durations to block heights', async () => {
```

Comment:
Most of the logic in `assembleScheduleRequest()` has been extracted to `getSubAuctions`; thus, so too have the tests.